src/app/Api/Dockerfile

```diff
@@ -18,4 +19,11 @@ RUN dotnet publish "Api.csproj" -c Release -o /app/publish
 FROM base AS final
 WORKDIR /app
 COPY --from=publish /app/publish .
-ENTRYPOINT ["dotnet", "Api.dll"]
\ No newline at end of file
+
+RUN apt-get update && apt-get install -y wget netcat-openbsd gnupg
+RUN wget -qO - https://www.mongodb.org/static/pgp/server-4.2.asc | apt-key add -
+RUN echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/4.2 multiverse" | tee /etc/apt/sources.list.d/mongodb-org-4.2.list
+RUN apt-get update && apt-get install -y mongodb-org=4.2.6

```

Comment:
Please verify if it's possible to use smaller package (for example, `mongodb-org-shell`):
https://docs.mongodb.com/manual/tutorial/install-mongodb-on-ubuntu/#mongodb-community-edition-packages